test/action_controller/json_api/linked_test.rb

```diff
@@ -43,29 +43,29 @@ def render_resource_without_include
 
           def render_resource_with_include
             setup_post
-            render json: @post, include: 'author', adapter: :json_api
+            render json: @post, include: [:author], adapter: :json_api
           end
 
           def render_resource_with_nested_include
             setup_post
-            render json: @post, include: 'comments.author', adapter: :json_api
+            render json: @post, include: [comments: [:author]], adapter: :json_api
```

Comment:
They broke when I removed the string syntax, and stopped breaking when I added it back.